src/components/Home/WalletDecryption/WalletDecryption.tsx

```diff
@@ -26,28 +26,25 @@ export const WalletDecryption: FunctionComponent<WalletDecryption> = ({
   };
   const isDecrypting = decryptProgress > 0 && decryptProgress < 1;
 
-  const inputBorderCSS = isIncorrectPassword
-    ? "w-full border-solid border border-red h-10 p-3"
-    : "w-full border-solid border border-gray-300 h-10 p-3";
+  let inputBorderCSS = `w-full border-solid border h-10 p-3 rounded-lg`;
+
+  inputBorderCSS += isIncorrectPassword ? ` border-red` : ` border-gray-300`;
```

Comment:
can change the red and gray to the new colors?

Reply:
red -> rose
gray-300 -> cloud-200?
double checking cos not specified on figma for this particular component